src/property-filter/styles.scss

```diff
@@ -43,21 +43,42 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  &-line {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
+  &-form {
+    display: grid;
+    grid-template-columns: max-content minmax(awsui.$space-xl, 1fr) minmax(200px, min-content);
+    grid-row-gap: awsui.$space-scaled-l;
+    margin-bottom: awsui.$space-scaled-l;
```

Comment:
minor: I can see that the `margin-bottom` was included in the previous code. However, maybe we can fix it. I can't imagine a good reason to have it atm.